pkgs/by-name/ca/cargo-bundle/package.nix

```diff
@@ -6,39 +6,63 @@
   stdenv,
   libxkbcommon,
   wayland,
+  openssl,
+  squashfsTools,
+  makeWrapper,
+  versionCheckHook,
+  nix-update-script,
 }:
 
-rustPlatform.buildRustPackage {
+rustPlatform.buildRustPackage (finalAttrs: {
   pname = "cargo-bundle";
-  # the latest stable release fails to build on darwin
-  version = "unstable-2023-08-18";
+  version = "0.9.0";
 
   src = fetchFromGitHub {
     owner = "burtonageo";
     repo = "cargo-bundle";
-    rev = "c9f7a182d233f0dc4ad84e10b1ffa0d44522ea43";
-    hash = "sha256-n+c83pmCvFdNRAlcadmcZvYj+IRqUYeE8CJVWWYbWDQ=";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-8Ulah5NtjQh5dIB/nhTrDstnaub4LS9iH33E1iv1JpY=";
   };
 
-  cargoHash = "sha256-g898Oelrk/ok52raTEDhgtQ9psc0PFHd/uNnk1QeXCs=";
+  cargoHash = "sha256-qE0ZDq0UJHfsivvI1W44u/pVjKMDGrghSl7sfau/pIY=";
 
   nativeBuildInputs = [
     pkg-config
+    makeWrapper
   ];
 
-  buildInputs = lib.optionals stdenv.hostPlatform.isLinux [
+  buildInputs = [
+    squashfsTools
+  ]
+  ++ lib.optionals stdenv.hostPlatform.isLinux [
     libxkbcommon
     wayland
+    openssl
   ];
 
-  meta = with lib; {
+  fixupPhase = ''
+    runHook preFixup
+
+    wrapProgram $out/bin/cargo-bundle \
```

Comment:
Shift this to `postFixup` instead of `fixupPhase`, that way you don't have to explicitly `runHook` the `pre*` and `post*` phases

Reply:
Done